GeometryCore/src/AABB.cpp

```diff
@@ -14,18 +14,36 @@ namespace geom_utils
         calculate(Polygon::makePolygon(tri));
     }
 
+    bool AABB2D::isValid(const AABB2D& aabb2d) 
+    {
+        if (aabb2d.pmax.x <= aabb2d.pmin.x || aabb2d.pmax.y <= aabb2d.pmin.y) return false;
```

Comment:
Imagine bound box for one 2D point. Bound box should be valid even for one point yet pmin is equal to pmax.
But with current code it won't be valid..